event/register_oracle_event.go

```diff
@@ -51,9 +59,93 @@ func (e RegisterOracleEvent) EventHandler(event ctypes.ResultEvent) error {
 
 	err = sgx.VerifyRemoteReport(oracleRegistration.NodePubKeyRemoteReport, nodePubKeyHash[:], *e.reactor.EnclaveInfo())
 	if err != nil {
+		msgVoteOracleRegistrationNo, err := makeOracleRegistrationVote(uniqueID, e.reactor.OracleAcc().GetAddress(), addressValue, types.VOTE_OPTION_NO, e.reactor.OraclePrivKey())
+		if err != nil {
+			return err
+		}
+
+		txBytes, err := generateTxBytes(msgVoteOracleRegistrationNo, e.reactor.OracleAcc().GetPrivKey(), e.reactor.Config(), e.reactor.TxBuilder())
+		if err != nil {
+			return err
+		}
+
+		err = broadCastTx(e.reactor.GRPCClient(), txBytes)
+		if err != nil {
+			return err
+		}
 		return err
```

Comment:
When the remote report is invalid, if `generatingTxBytes` and `broadcastTx` are succeed, it should return `nil`, not `err`. Because it is a normal case of invalid oracle registration vote.
```suggestion
		return nil
```

Reply:
Since this error occurred while executing generateTxBytes() or broadCastTx(), wouldn't it be appropriate to return an error?

Reply:
Right. However the error occurred from `generateTxBytes()` and `broadcastTx()` is checked if it is nil or not already.
If there is no error, I think, we should `return nil` even if the remote report is invalid.

Reply:
Oh, I understand. In the current structure, we would vote No and then vote Yes again.